lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -98,9 +106,9 @@ def test_course_access_to_beta_users(self):
         beta_tester = BetaTesterFactory(course_key=self_paced_course.id)
 
         # Verify course is `self_paced` and course has start date but not section.
-        self.assertTrue(self_paced_course.self_paced, "Course is self_paced")
-        self.assertEqual(self_paced_course.start, one_month_from_now, "Course has start date")
-        self.assertIsNone(self_paced_section.start, "Section start date is None")
+        self.assertTrue(self_paced_course.self_paced)
+        self.assertEqual(self_paced_course.start, one_month_from_now)
+        self.assertIsNone(self_paced_section.start)
```

Comment:
Because:
1. They didn't add any information above what the default error message would tell us.
2. They were lies.  Notice that they state the positive condition from the assertion, but get printed out when the negative condition happens.  I discovered this when, due to the test bug this PR exposed, I got the error "Section start date is None".  Debugging eventually demonstrated that the error was that the start_date was *not* None.